src/Engine/ProtoScript/Utils/ProtoScriptUtils.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using ProtoCore;
+
+namespace ProtoScript.Utils
+{
+    public static class ProtoScriptUtils
+    {
+        public static ProtoCore.Core CreateCore(Options coreOptions,ProtoCore.Core runnerCore)
```

Comment:
@lukechurch Sure ill ping you about this. the intention was to try having a common function that creates a Core instead of manually calling its constructors and setting members